docs/faq.md

```diff
@@ -188,11 +188,11 @@ is complete before continuing.
 How do I switch off an option in the CLI?
 -----------------------------------------
 i.e. `webdriver-manager update --chrome=false` does not work. 
-This has to do with the way `optimist` parses command line args. In order to pass a false value, do one of the following:
+This has to do with the way `yargs` parses command line args. In order to pass a false value, do one of the following:
```

Comment:
Is this still the case with current `yargs`? Can it parse `--chrome=false` correctly?

Reply:
Good catch, that should be addressed. I’ll remove this section

Reply:
Updated.